APP_DEVELOPMENT.md

```diff
@@ -82,6 +82,9 @@ There are a few steps to add and use an environment variable using Expo:
 3. In the file in your code where you want to use this variable, add an import statement at the top of the file `import Constants from 'expo-constants'`
 4. In the same file, at the place in the code where you want to use the variable, you can access your variable with e.g. `Constants.expoConfig?.extra?.amazingApiKey` (note: Typescript considers that the value of this could be `undefined`, so you may need to handle that possibility)
 
+### Version
+API version 1.1 are protected by API_KEY, that needs to be configured in `api/.env` and `app/.env` file. The version to be used in the development can be changed by modifying the STA_API_VERSION in the `app/.env` file in the `app` directory.
```

Comment:
It looks like this is named `API_KEY` in the API and `STA_API_KEY` in the app -- suggest it might be even better if it's standardised so it's the same everywhere (`STA_API_KEY` seems clearest, to differentiate it from other services), this will probably help avoid confusion so devs put the same line in both their .env files